game-core/src/main/java/games/strategy/engine/framework/startup/ui/InGameLobbyWatcher.java

```diff
@@ -218,11 +218,13 @@ public static InGameLobbyWatcher newInGameLobbyWatcher(
               IServerMessenger.getRealName(hostedBy) + "_" + LobbyConstants.LOBBY_WATCHER_NAME,
               mac,
               login);
-      final UnifiedMessenger um = new UnifiedMessenger(messenger);
-      final RemoteMessenger rm = new RemoteMessenger(um);
-      final RemoteHostUtils rhu = new RemoteHostUtils(messenger.getServerNode(), gameMessenger);
-      rm.registerRemote(rhu, IRemoteHostUtils.Companion.newRemoteNameForNode(um.getLocalNode()));
-      return new InGameLobbyWatcher(messenger, rm, gameMessenger, handler, oldWatcher);
+      final var unifiedMessenger = new UnifiedMessenger(messenger);
+      final var remoteMessenger = new RemoteMessenger(unifiedMessenger);
+      final var remoteHostUtils = new RemoteHostUtils(messenger.getServerNode(), gameMessenger);
```

Comment:
Excellent use of the var keyword 👍

Reply:
Thanks! It's not universally a good thing, but this does seem to be a really good example of where it's useful.